app/controllers/admin/radios_controller.rb

```diff
@@ -0,0 +1,74 @@
+class Admin::RadiosController < ApplicationController
+  before_action :set_radio, only: [:show, :edit, :update, :destroy]
+
+  # GET /radios
+  # GET /radios.json
+  def index
+    @radios = Radio.all
+  end
+
+  # GET /radios/1
+  # GET /radios/1.json
+  def show
+  end
+
+  # GET /radios/new
+  def new
+    @radio = Radio.new
+  end
+
+  # GET /radios/1/edit
+  def edit
+  end
+
+  # POST /radios
+  # POST /radios.json
+  def create
+    @radio = Radio.new(radio_params)
+
+    respond_to do |format|
+      if @radio.save
+        format.html { redirect_to @radio, notice: 'Radio was successfully created.' }
+        format.json { render :show, status: :created, location: @radio }
+      else
+        format.html { render :new }
+        format.json { render json: @radio.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /radios/1
+  # PATCH/PUT /radios/1.json
+  def update
+    respond_to do |format|
+      if @radio.update(radio_params)
+        format.html { redirect_to @radio, notice: 'Radio was successfully updated.' }
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->